omim2obo/main.py

```diff
@@ -197,28 +199,27 @@ def omim2obo(use_cache: bool = False):
         else:
             pass
 
-        if use_abbrev_over_label and abbrev:
-            graph.add((omim_uri, RDFS.label, Literal(abbrev)))
+        if use_abbrev_over_label and pref_symbols:
+            if len(pref_symbols) > 1:
```

Comment:
There is a section that describes the file format for "Entry Data" on https://omim.org/help/api#1_1. It describes the "preferredTitle" as "title and symbol, delimited with ';'". I analyzed the mimTitles.txt file from 4-Sep-2024 and there are 2 entries where there is >1 symbol in this field.

These are: 

MIM Number | Preferred Title; symbol
-- | --
126370 | DNA, SATELLITE, III; HS3; D1Z1
171820 | PHOSPHATASE, SALIVARY ACID, A; SACP; ACPS

Therefore, for the "preferredTitle" field, anything after the first `;` can be considered a symbol.

Keep in mind "includedTitles" are formatted differently so what is decided for the "preferredTitle" field should not be confused with what needs to happen for values in the "includedTitles" field.

Reply:
That corroborates my reading of the docs and analysis as well (described a little in #125).

> Therefore, for the "preferredTitle" field, anything after the first ; can be considered a symbol.

I believe the code does that correctly now!

> Keep in mind "includedTitles" are formatted differently

Yeah, I wasn't sure. That issue (#118), I'll either do in the greater #128 PR or a separate PR.

Reply:
The script should not fail when running if >1 abbreviation is found in the "preferredTitle" field.

Reply:
Ah. Yes, that is my current failsafe until we decide what to do about these cases.

It doesn't just fail if there is more than 1 abbreviation. This is a more complex case.

I've only set it to fail if we are using the symbol/abbreviation as the main `rdfs:label` rather than a Synonym.

The reason for that is because if we want to use an symbol as a label, I don't know yet which symbol to pick if there are >1, or whether I should just include all of them as a concatenated string.

The only situation where we do this; set the abbrev as a label, is if `omim_type == OmimType.GENE or omim_type == OmimType.HAS_AFFECTED_FEATURE:  # * or +`.

Note that there are no current cases where this is failing in the code.

Reply:
Thanks for the explanation, since this check is only when `omim_type == OmimType.GENE or omim_type == OmimType.HAS_AFFECTED_FEATURE:  # * or +` is in place then it's ok to leave this.

Reply:
@twhetzel I'm surprised! `mimTitles.txt` was just updated, and a MIM is now triggering this error!

[OMIM:171820](https://omim.org/entry/171820): _PHOSPHATASE, SALIVARY ACID, A; SACP; ACPS_

I made these changes:
* Warning now instead of error
* Picks the first symbol to use as label

I don't know how concerned we have to be about this, because these gene classes don't really get into Mondo anyway.